spring-batch-docs/asciidoc/repeat.adoc

```diff
@@ -75,31 +74,31 @@ template.iterate(new RepeatCallback() {
 });
 ----
 
-In the example we return `RepeatStatus.CONTINUABLE` to
+In the preceding example, we return `RepeatStatus.CONTINUABLE`, to
     show that there is more work to do. The callback can also return
-    `RepeatStatus.FINISHED` if it wants to signal to the caller that
+    `RepeatStatus.FINISHED`, to signal to the caller that
     there is no more work to do. Some iterations can be terminated by
-    considerations intrinsic to the work being done in the callback, others
+    considerations intrinsic to the work being done in the callback. Others
     are effectively infinite loops as far as the callback is concerned and the
-    completion decision is delegated to an external policy as in the case
-    above.
+    completion decision is delegated to an external policy, as in the case
+    shown in the preceding example.
 
 [[repeatContext]]
 
 
 ==== RepeatContext
 
 The method parameter for the `RepeatCallback`
-      is a `RepeatContext`. Many callbacks will simply
-      ignore the context, but if necessary it can be used as an attribute bag
+      is a `RepeatContext`. Many callbacks
+      ignore the context. However, if necessary, it can be used as an attribute bag
       to store transient data for the duration of the iteration. After the
-      `iterate` method returns, the context will no
-      longer exist.
+      `iterate` method returns, the context no
+      longer exists.
 
-A `RepeatContext` will have a parent context
-      if there is a nested iteration in progress. The parent context is
+if there is a nested iteration in progress, a `RepeatContext` has a parent context.
```

Comment:
If needs to be capitalized.  If you agree I can fix on merge.

Reply:
Yes, please. Thanks, Glenn.